packages/create-cloudflare/e2e-tests/workers.test.ts

```diff
@@ -197,18 +198,20 @@ describe
 							const tomlPath = join(project.path, "wrangler.toml");
 							const jsoncPath = join(project.path, "wrangler.jsonc");
 
-							try {
-								expect(jsoncPath).toExist();
+							if (existsSync(jsoncPath)) {
 								const config = readJSON(jsoncPath) as { main?: string };
 								if (config.main) {
 									expect(join(project.path, config.main)).toExist();
 								}
-							} catch (e) {
-								expect(tomlPath).toExist();
+							} else if (existsSync(tomlPath)) {
 								const config = readToml(tomlPath) as { main?: string };
 								if (config.main) {
 									expect(join(project.path, config.main)).toExist();
 								}
+							} else {
+								expect.fail(
+									`Expected at least one of "${jsoncPath}" and "${tomlPath}" to exist.`,
```

Comment:
```suggestion
									`Expected at least one of "${jsoncPath}" or "${tomlPath}" to exist.`,
```

I think?

Reply:
I think either is correct enough. But since you asked...